src/net.h

```diff
@@ -505,6 +505,10 @@ class CNode
      * peer eviction criterium in CConnman::AttemptToEvictConnection. */
     std::atomic<std::chrono::seconds> m_last_block_time{0s};
 
+    /** UNIX epoch time of the last block announcement from this peer that we had
+     * not yet seen from any peer. Used for potential stale-tip eviction. */
+    std::atomic<std::chrono::seconds> m_last_block_announcement{0s};
```

Comment:
If you change the type, why not use NodeSeconds (a time point)? While `std::chrono::seconds` (a duration) is used widely in the codebase, because all durations that denote a time point are since epoch, using a dedicated type such as `NodeSeconds` might be clearer.